ufl/operators.py

```diff
@@ -159,7 +179,7 @@ def dot(a, b):
     a = as_ufl(a)
     b = as_ufl(b)
     if a.ufl_shape == () and b.ufl_shape == ():
-        return a*b
+        return a*Conj(b)
```

Comment:
docstrings should document the convention of insertion of conjugate.

Reply:
Done.